toolkit/tools/imagegen/diskutils/diskutils.go

```diff
@@ -379,7 +379,7 @@ func WaitForLoopbackToDetach(devicePath string, diskPath string) error {
 	}
 
 	delay := 100 * time.Millisecond
-	attempts := 5
+	attempts := 10
```

Comment:
This sets the total timeout to 102 seconds, which might be a tad long. I think 8 loops for a total of 25 seconds might be a bit more reasonable.